### PR TITLE
Fix: withdrawals wait time text fix

### LINKED ISSUE #516: Invalid time in the WITHDRAWALS-LONG-UNFINALIZED-QUEUE alert header
Screenshot 2024-04-08 at 09 48 41

Wait time in the alert body and in the header are different

### PR DESCRIPTION
**Pull request by TheDZhon**
closes #516 

- [x] Update alert text
- [x] Add test

### CODE REVIEW

File: ethereum-steth/src/services/withdrawals/Withdrawals.srv.ts [hunk 1]
```diff
@@ -356,17 +356,17 @@ export class WithdrawalsSrv {
...
 
-        if (waitTime > LONG_UNFINALIZED_QUEUE_TRIGGER_EVERY) {
+        if (timeSinceLastAlert > LONG_UNFINALIZED_QUEUE_TRIGGER_EVERY) {
           // if we are in turbo mode and unfinalized queue is not finalized for 5 days
           // and alert hasn't been sent for 1 day
           out.push(
             Finding.fromObject({
-              name: `⚠️ Withdrawals: unfinalized queue wait time is ${new Date(waitTime * 1000).getHours()} hr`,
-              description: `Unfinalized queue wait time is ${formatDelay(
+              name: `⚠️ Withdrawals: unfinalized queue wait time is more than ${LONG_UNFINALIZED_QUEUE_THRESHOLD / ONE_DAY} days`,
```
Comment (sergeyWh1te) [inline ethereum-steth/src/services/withdrawals/Withdrawals.srv.ts:366]: Let's rename LONG_UNFINALIZED_QUEUE_THRESHOLD  -> DAYS_5

File: ethereum-steth/src/services/withdrawals/Withdrawals.srv.ts [hunk 2]
```diff
@@ -356,17 +356,17 @@ export class WithdrawalsSrv {
 
     if (!this.cache.getIsBunkerMode() && unfinalizedStETH.gt(0)) {
       if (currentBlockTimestamp - LONG_UNFINALIZED_QUEUE_THRESHOLD > firstUnfinalizedRequest.right.timestamp) {
-        const waitTime = currentBlockTimestamp - this.cache.getLastLongUnfinalizedQueueAlertTimestamp()
+        const timeSinceLastAlert = currentBlockTimestamp - this.cache.getLastLongUnfinalizedQueueAlertTimestamp()
 
-        if (waitTime > LONG_UNFINALIZED_QUEUE_TRIGGER_EVERY) {
+        if (timeSinceLastAlert > LONG_UNFINALIZED_QUEUE_TRIGGER_EVERY) {
```
Comment (sergeyWh1te) [inline ethereum-steth/src/services/withdrawals/Withdrawals.srv.ts:361]: if (timeSinceLastAlert > DAY_1)

**Review by sergeyWh1te — CHANGES_REQUESTED** [review]

File: ethereum-steth/src/services/withdrawals/Withdrawals.srv.ts (hunk 1 above)
Comment (TheDZhon) [reply ethereum-steth/src/services/withdrawals/Withdrawals.srv.ts]: Fixed

File: ethereum-steth/src/services/withdrawals/Withdrawals.srv.ts (hunk 2 above)
Comment (TheDZhon) [reply ethereum-steth/src/services/withdrawals/Withdrawals.srv.ts]: Fixed

**Review by sergeyWh1te — APPROVED** [review]

**Pull request merged**